api/e2e/test/helpers_api_test.go

```diff
@@ -41,7 +41,7 @@ func getRouter(
 
 	err = json.Unmarshal(respBytes, &router)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("Could not unmarshal: %v\n %s", err, string(respBytes))
```

Comment:
Added this for debugging e2e test failure, but I think it's ok to keep it.

![Screenshot 2022-04-27 at 11 27 43 AM](https://user-images.githubusercontent.com/23465343/165437189-4cfe1038-d948-4341-959f-0ebe4c13b3b3.png)